packages/block-editor/src/components/block-list/style.scss

```diff
@@ -19,6 +19,9 @@
 	}
 }
 
+.block-editor-block-styles .editor-styles-wrapper .block-editor-block-list__block {
```

Comment:
A comment explaining why this style is here would be helpful to others working on this code in the future.

Reply:
Should this be moved to the `block-editor-block-styles` style file instead?

Reply:
Also did we test with blocks that have "inner blocks"?

Reply:
Happy to move this to `block-editor-block-styles` but I thought it was better to live near the specific code its targeting. 

Is there an example of a block with inner blocks which has a style preview?

Reply:
> Is there an example of a block with inner blocks which has a style preview?

Yes, the columns, group, media & text blocks.

It seems like there's something going wrong with the blocks you're trying to preview. Should we dig deeper to try to understand why it doesn't behave as expected?

Reply:
I don't see style previews for these blocks in the side bar.

Reply:
Right! they only have an example shown in the inserter, to trigger the sidebar, you can just register a fake style variation.